File: shoppy-backend/src/main/java/com/app/shoppybackend/entity/CartItem.java
package com.app.shoppybackend.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "cart")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class CartItem {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String userEmail;

    @ManyToOne
    @JoinColumn(name = "product_id")
    private Product product;

    private String selectedSize;
    
    private Integer quantity;
    
    private Double price;
}

File: app/src/main/java/com/app/stylenest/model/CartItem.kt
package com.app.stylenest.model

data class CartItem(
    val id: Int = 0,
    val productId: Int,
    val selectedSize: String,
    var quantity: Int,
    val productName: String = "",
    val price: Double = 0.0,
    val imageName: String = ""
)
